sysutils/spacebar/Portfile

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               makefile 1.0
+PortGroup               xcodeversion 1.0
+
+github.setup            cmacrae spacebar 1.4.0 v
+
+categories              sysutils
+maintainers             {@bashu gmail.com:bashu.was.here} openmaintainer
+license                 MIT
+description             A minimal status bar for macOS.
+long_description        spacebar is a minimal status bar for macOS. \
+                        Ideal for use with tiling window managers like yabai.
+
+checksums               rmd160  f73ec0cd74e86bc913ac0c55ad9839a7766220f9 \
+                        sha256  6e89a921fd100d8e61140a0457ce9feab9cde9480bb780e03a215e5c5154793e \
+                        size    44951
+
+patchfiles              patch-makefile.diff patch-src__event_loop.c.diff
+
+set spacebar.plist      ${workpath}/org.macports.spacebar.plist
+
+post-patch {
+    # Fill in startup plist template
+    copy -- ${filespath}/org.macports.spacebar.plist.in ${spacebar.plist}
+    reinplace "s|@PREFIX@|${prefix}|g" ${spacebar.plist}
+}
+
+post-destroot {
+    # Copy manpage
+    # TODO: build manpage properly
+    xinstall -m 444 ${worksrcpath}/doc/spacebar.1 ${destroot}${prefix}/share/man/man1
+
+    # Copy example files
+    xinstall -d -m 0755 ${destroot}${prefix}/share/doc/${name}/examples
+    copy {*}[glob ${worksrcpath}/examples/*] \
+        ${destroot}${prefix}/share/doc/${name}/examples
+
+    # Install our custom plist
+    set launchd_dir ${prefix}/etc/${startupitem.location}/${startupitem.uniquename}
+    xinstall -m 755 -d ${destroot}/${launchd_dir}
+    xinstall -m 644 ${spacebar.plist} ${destroot}/${launchd_dir}/${startupitem.plist}
+
+    if {[getuid] == 0 && ${startupitem.install} ne "no"} {
+        file mkdir ${destroot}/Library/${startupitem.location}
+        ln -sf ${launchd_dir}/${startupitem.plist} ${destroot}/Library/${startupitem.location}
+    } else {
+        ln -sf ${launchd_dir}/${startupitem.plist} ${destroot}${prefix}/etc/${startupitem.location}
+    }
+}
+
+set minxcodever 8.0
+if {([vercmp $xcodeversion ${minxcodever}] < 0)} {
+    known_fail yes
+    pre-fetch {
+        ui_error "${name} ${version} requires Xcode ${minxcodever} or greater to build."
+        return -code error "incompatible Xcode version"
+    }
+}
```

Comment:
I don't think this is how the test is supposed to work, but am not 100% sure. You do add the `xcodeversion` PortGroup at the top of the file but are not using it... Are you not supposed to just use `minimum_xcodeversions   {darwin_major minimum_xcodeversion}` (see [here](https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/xcodeversion-1.0.tcl))?

Reply:
fixed